src/EventListener/InitializeSystemListener.php

```diff
@@ -138,10 +138,12 @@ public function onConsoleCommand()
     protected function setConstants($mode, $route)
     {
         // The constants are deprecated and will be removed in version 5.0.
+        $time = microtime(true);
```

Comment:
But should be.
IMO we should deprecate every constant to encourage writing properly testable code.
Constants not be used anymore not by legacy extensions and not even in the core-bundle but this will take a migration phase during the 4.x series.
To allow this, we should deprecate them all as of now.